l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1036,6 +1036,81 @@ def strip_xml_content(self, xml):
                 elem.text = elem.text.strip()
         return etree.tostring(root)
 
+    def check_file_is_pem(self, p7m_file):
+        file_is_pem = True
+        strcmd = (
+            'openssl asn1parse  -inform PEM -in %s'
+        ) % (p7m_file)
+        cmd = shlex.split(strcmd)
+        try:
+            proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+            stdoutdata, stderrdata = proc.communicate()
+            if proc.wait() != 0:
+                file_is_pem = False
+        except Exception as e:
+            raise orm.except_orm(
+                _('Errore'),
+                _(
+                    'Check PEM file %s'
+                ) % e.args
+            )
+        return file_is_pem
+
+    def parse_pem_2_der(self, pem_file, tmp_der_file):
+        strcmd = (
+            'openssl asn1parse -in %s -out %s'
+        ) % (pem_file, tmp_der_file)
+        cmd = shlex.split(strcmd)
+        try:
+            proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+            stdoutdata, stderrdata = proc.communicate()
+            if proc.wait() != 0:
+                _logger.warning(stdoutdata)
+                raise Exception(stderrdata)
+            if not os.path.isfile(tmp_der_file):
+                raise orm.except_orm(
```

Comment:
in realta' e' sbagliata la posizione sia dell' if che del orm.exception
giustamente come fai otare c'e' il try, quindi  tutti gli errori all interno sono gestiti dalla 1077, a questo punto sposterei 
fuori dal try la verifica del file in modo da specificare il tipo di anomalia.